src/fides/api/main.py

```diff
@@ -81,6 +86,18 @@
     view.router,
 ]
 
+if CONFIG.security.env.value == "dev":
```

Comment:
Some explicit logging in these blocks would go a long way for debugging

Reply:
Wouldn't that be part of the `config` though? Since these values are pulled directly from the config, I think looking at the config is enough?